src/contributing/void-docs/submitting.md

```diff
@@ -0,0 +1,51 @@
+## Submitting Changes
+
+### Requirements
+
+To clone the repository and push changes
+[git(1)](https://man.voidlinux.org/git.1) is required, which is available as the
+`git` package.
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/), which can be installed
+with the `mdBook` package on Void. At the root of the void-docs repository
+`mdbook serve` can be run to serve the docs on your localhost.
+
+### Forking
+
+To fork the repository a [github account](https://github.com/join) is needed.
+After the account is created follow github's
+[guide](https://help.github.com/en/articles/fork-a-repo) on setting up a fork.
+
+Clone the repository onto your computer, enter it, and create a new branch:
+
+```
+$ git clone https://github.com/YOUR_USERNAME/void-docs.git
+$ cd void-docs
+$ git checkout -b <BRANCH_NAME>
+```
+
+> The branch name should be based off of the planned change.
+
+After editing the file(s), commit the changes and push them to the forked
+repository:
+
+```
+$ git add <EDITED_FILE(S)>
+$ git commit -m "<COMMIT_MESSAGE>"
+$ git push --set-upstream origin <BRANCH_NAME>
+```
+
+> The commit message should be in the form of `something: reduced section x and
```

Comment:
sorry to add one last thing, but I think this would be clearer just saying `'section: what was changed'` or something like that.